6.x/crud-uploaders.md

```diff
@@ -0,0 +1,97 @@
+# Uploaders
+
+---
+
+<a name="upload-about"></a>
+## About
+
+Uploading and managing files is a common task in Admin Panels. Starting with Backpack v6, you can fully setup your upload fields in your field definition. No more mutators, manual validation of input, or custom code to handle the files.
+
+<a name="upload-how-it-works"></a>
+## How it works
+
+When adding an upload field (`upload`, `upload_multiple`, `image` or `dropzone`) to your operations, you can now tell Backpack that you want it to upload those files to a specific disk, path and customize some aditional things. Get to know `withFiles()`. 
+    
+```php
+CRUD::field('avatar')
+    ->type('upload')
+    ->withFiles();
+```
+> **IMPORTANT**: make sure you linked the `storage` folder to your `public` folder. You can do that by running `php artisan storage:link` in your terminal.
+
+That's it. Backpack will now handle the upload, storage and deletion of the files for you. By default it will use `public` disk, and will delete the files when the entry is deleted.
+
+<a name="upload-configuration"></a>
+## Configuring the uploads
+
+The `withFiles()` method accepts an array of options that you can use to customize the upload. 
+
+```php
+CRUD::field('avatar')
+    ->type('upload')
+    ->withFiles([
+        'disk' => 'public', // the disk where file will be stored
+        'path' => 'uploads', // the path inside the disk where file will be stored
+        'deleteWhenEntryIsDeleted' => true, // if the files should be deleted when entry is deleted
+
+        // cloud disks like s3 allow the usage of temporary urls
+        'temporary' => false, 
+        'expiration' => 1 // the corresponding temporary url expiration time in minutes
+]);
+```
+**Note**: `disk` and `prefix` (called `path` in withFiles() configuration), when defined in the field as attributes, will take precedence over the definition in `withFiles()` configuration. 
+Make sure you are not defining those attributes in both places.
```

Comment:
```suggestion
**Note**: If you've defined `disk` or `prefix` on the field, you no longer need to define `path` within `withFiles()` - it will pick those up. Make sure you are not defining both.
```